app/controllers/applications_controller.rb

```diff
@@ -23,12 +20,24 @@ def create
       redirect_to "/applications/new"
     end
   end
-
+  
   def update
     application = Application.find(params[:id])
     application.update(status: 1)
     application.update(good_owner: params[:good_owner])
     redirect_to "/applications/#{application.id}"
   end
+
+  def search_by_name(name)
+    Pet.search(name) if name.present?
```

Comment:
if we change the above comment then we will have to change name to name[:search). But that might look weird if that line looks like `Pet.search(name[:search]) if name[:search].present?`